snap/validate.go

```diff
@@ -351,6 +351,13 @@ func validateAppOrderNames(app *AppInfo, dependencies []string) error {
 	return nil
 }
 
+func validateAppWatchdog(app *AppInfo) error {
+	if app.WatchdogTimeout != 0 && !app.IsService() {
+		return fmt.Errorf("cannot use watchdog, application %q is not a service", app.Name)
```

Comment:
The comma seems misplaced here, and we already have a pattern above reporting the exact same issue, so it's best to use the exact same wording too:

_"cannot define watchdog-timeout in application %q as it's not a service"_

Alternatively, we can change both to a third form, but doesn't seem necessary for this case.